2-mandatory/2-water-bottle.js

```diff
@@ -19,18 +19,27 @@ let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    this.volume = 100
   },
   pour: function () {
     // calling this function should increase your bottle volume by 10 units;
+   if (this.volume < 100 ) {
+    return this.volume += 10 ;
```

Comment:
This is a good start! But have you considered what would happen if the volume was at something like 97 and you added 10 to it considering 100 is the max?